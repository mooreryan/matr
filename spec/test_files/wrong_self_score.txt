source	target	score
ryan	ryan	1
ryan	amelia	101