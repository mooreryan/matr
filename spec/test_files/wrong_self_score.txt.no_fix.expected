	amelia	ryan
amelia	100	101
ryan	101	1
